code/game/gamemodes/cult/runes.dm

```diff
@@ -190,6 +194,9 @@ structure_check() searches for nearby cultist structures required for the invoca
 			L.apply_damage(invoke_damage, BRUTE)
 			to_chat(L, "<span class='cultitalic'>[src] saps your strength!</span>")
 	do_invoke_glow()
+	SSblackbox.record_feedback("nested tally", "runes_invoked", 1, list("[initial(cultist_name)]", "[length(SSticker.mode.cult)]")) // the name of the rune, and the number of cultists in the cult when it was invoked
+	if(ghost_invokers)
+		SSblackbox.record_feedback("nested tally", "runes_invoked_with_ghost", 1, list("[initial(cultist_name)]", "[ghost_invokers]")) //the name of the rune and the number of ghosts used to invoke it.
```

Comment:
I worry about this skewing data a bit as this logs 1 invoke as 2 separate things. I would rather you look at how the nested tallies for end round stats (players alive, players on pod, etc) so you can log more info per invoke, but maybe thats outside the scope. Not 100% sure.

Reply:
I think we might run into issues with data representation, currently with nested tallies you record 2 sets of data + the key. 
so you can plot x against y. 
if you go higher than that it gets messy. Also, it doesn't exactly skew data as we can find the "runes_invoked_with_ghost" / "runes_invoked" to find the proportion of runes invoked with a ghost.

Reply:
True, I guess since we can just use the normal invoked number